ftfy/__init__.py

```diff
@@ -161,65 +186,73 @@ def fix_text(text,
     return ''.join(out)
 
 ftfy = fix_text
+fix_encoding = fixes.fix_encoding
```

Comment:
Do you think it's okay for `fix_encoding` the parameter and `fix_encoding` the top-level function to have the same name, if I fix the other inconsistencies?

I probably called it `fix_text_encoding` before in part to avoid this parameter collision, but I want the function to have a short, accessible name because some use cases should just go straight to `fix_encoding`, not `fix_text`.